pkg/lib/patternfly/patternfly-4-overrides.scss

```diff
@@ -28,6 +28,21 @@ ul.pf-c-select__menu {
     overflow-y: auto;
 }
 
+/* Adjust padding on form selects to resemble PF non-form selects */
+/* (This can be seen when the longest text is selected on a non-stretched select) */
+/* Upstream: https://github.com/patternfly/patternfly/issues/4387 */
+/* Cockpit-Podman: https://github.com/cockpit-project/cockpit-podman/issues/755 */
+select.pf-c-form-control {
+    --pf-c-form-control--PaddingRight: 41px;
+    --pf-c-form-control--PaddingLeft: 6px;
```

Comment:
We could possibly reverse out the padding to the normal selects, if they apply the values on the `pf-c-form-control` and use that minus some values (possibly spacing related, but as this doesn't divide cleanly, we would probably still have to add magic `px` values, especially to the Firefox compensation below).

In other words, I think we're stuck with magic values in `px` one way or the other, to make form selects look more like custom selects.